<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCircuitCfgForm</class>
 <widget class="QWidget" name="CCircuitCfgForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>350</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Obwody sterowania</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QFrame" name="cirRelayFrame">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>105</y>
     <width>215</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_46">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>170</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sterowanie stycznikem:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_48">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr wyjścia:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cirRelIOMod">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="cirRelBitNo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>55</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="cirGroupNo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>105</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="cirWeatherAutomNo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>135</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="cirActiveChk">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>42</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Aktywny</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="l2Frame">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>215</width>
     <height>205</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="l2ActiveChk">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Faza L2</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="l2ConfActiveChk">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>170</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Potw. zabezpieczenia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>55</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="l2ConfIOMod">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="l2ConfBitNo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>85</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr wejścia:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>130</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>P2 nominalna [kW]:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tolerancja [kW]:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="p2Nominal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="p2Tol">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="l2CvmCh">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>40</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="l2CvmId">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>110</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr CVM:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_54">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>115</y>
      <width>40</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Kanał:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_59">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>130</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nr autom. pogod.:</string>
   </property>
  </widget>
  <widget class="QFrame" name="l1Frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>215</width>
     <height>205</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="l1ActiveChk">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Faza L1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="l1ConfActiveChk">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>170</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Potw. zabezpieczenia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>55</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="l1ConfIOMod">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="l1ConfBitNo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>85</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr wejścia:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>130</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>P1 nominalna [kW]:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tolerancja [kW]:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="p1Nominal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="p1Tol">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="l1CvmCh">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>40</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_57">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr CVM:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="l1CvmId">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>110</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_56">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>115</y>
      <width>40</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Kanał:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_58">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nr grupy:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>50</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa:</string>
   </property>
  </widget>
  <widget class="QFrame" name="l3Frame">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>215</width>
     <height>205</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="l3ActiveChk">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Faza L3</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="l3ConfActiveChk">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>170</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Potw. zabezpieczenia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>55</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="l3ConfIOMod">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="l3ConfBitNo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>85</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr wejścia:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>145</y>
      <width>130</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>P3 nominalna [kW]:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tolerancja [kW]:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="p3Nominal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="p3Tol">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_52">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr CVM:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="l3CvmId">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>110</y>
      <width>50</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_53">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>115</y>
      <width>40</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Kanał:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="l3CvmCh">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>40</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cirReferenceChk">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>42</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Wzorcowy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>45</y>
     <width>90</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Typ obwodu:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cirTypeCmb">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>40</y>
     <width>180</width>
     <height>27</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Ogrzewanie rozjazdów</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ogrzewanie zamknięć</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="cirNameEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>7</number>
   </property>
  </widget>
  <widget class="QFrame" name="cirRelayConfFrame">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>105</y>
     <width>215</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_49">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>170</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Potwierdzenie stycznika:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr modułu IO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_51">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nr wejścia:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cirRelConfIOMod">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="cirRelConfBitNo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>55</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>40</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Opis:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cirDescriptionEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>626</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>127</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
